pxtlib/blocks.ts

```diff
@@ -408,6 +410,8 @@ namespace pxt.blocks {
                     "ceil": Util.lf("{id:op}ceiling"),
                     "floor": Util.lf("{id:op}floor"),
                     "trunc": Util.lf("{id:op}truncate"),
+                    "idiv": Util.lf("{id:op}integer ÷"),
+                    "imul": Util.lf("{id:op}integer ×"),
```

Comment:
We seem to be using `/` and `*` in Math blocks - will this appear as `[10] integer ÷ [3]` ?

Reply:
Where? We use the symbols in the arithmetic blocks:

![image](https://user-images.githubusercontent.com/13754588/43985389-f80fe840-9cbb-11e8-8279-bc8e1cb04cbe.png)

And right now it looks like this:

![image](https://user-images.githubusercontent.com/13754588/43985422-3398f37a-9cbc-11e8-9b53-76d98b954621.png)

I'll try and move it to the center because I agree that would be much better

Reply:
Whoops, I was looking in TypeScript toolbox. Never mind.